docs/sponsors.rst

```diff
@@ -0,0 +1,33 @@
+Sponsors
```

Comment:
```suggestion
:hide-toc:

Sponsors
```

This'll look nicer.

Reply:
@sethmlarson *nudge*

Reply:
Thank you for the nudge! I need to implement this in my sponsor + opencollective branch.